index.html

```diff
@@ -0,0 +1,169 @@
+            <div class="card-text">
+                <h6>CANOPY</h6>
+                <ul>
+                    <li>Back End Dev</li>
+                    <li>2015</li>
+                </ul>
+            </div>
+            <p class="para">
+                A daily selection of privately personalized reads; no accounts or sign-ups required.
+            </p>
+            <ul>
+                <li class="list">HTML</li>
+                <li class="list">CSS</li>
+                <li class="list">JAVASCRIPT</li>
+            </ul>
+            <button class="btn" type="button">See Project</button>
+        </div>
+        <div class="cards">
+            <img src="images/card2.png" alt="">
+            <h1 class="card-title">Multi-Post Stories</h1>
+            <div class="card-text">
+                <h6>CANOPY</h6>
+                <ul>
+                    <li>Back End Dev</li>
+                    <li>2015</li>
+                </ul>
+            </div>
+            <p class="para">
+                A daily selection of privately personalized reads; no accounts or sign-ups required.
+            </p>
+            <ul>
+                <li class="list">HTML</li>
+                <li class="list">CSS</li>
+                <li class="list">JAVASCRIPT</li>
+            </ul>
+            <button class="btn" type="button">See Project</button>
+        </div>
+        <div class="cards">
+            <img src="images/card3.png" alt="">
+            <h1 class="card-title">Multi-Post Stories</h1>
+            <div class="card-text">
+                <h6>CANOPY</h6>
+                <ul>
+                    <li>Back End Dev</li>
+                    <li>2015</li>
+                </ul>
+            </div>
+            <p class="para">
+                A daily selection of privately personalized reads; no accounts or sign-ups required.
+            </p>
+            <ul>
+                <li class="list">HTML</li>
+                <li class="list">CSS</li>
+                <li class="list">JAVASCRIPT</li>
+            </ul>
+            <button class="btn">See Project</button>
+        </div>
+
+        <div class="cards">
+            <img src="images/card4.png" alt="">
+            <h1 class="card-title">Multi-Post Stories</h1>
+            <div class="card-text">
+                <h6>CANOPY</h6>
+                <ul>
+                    <li>Back End Dev</li>
+                    <li>2015</li>
+                </ul>
+            </div>
+            <p class="para">
+                A daily selection of privately personalized reads; no accounts or sign-ups required.
+            </p>
+            <ul>
+                <li class="list">HTML</li>
+                <li class="list">CSS</li>
+                <li class="list">JAVASCRIPT</li>
+            </ul>
+            <button class="btn">See Project</button>
+        </div>
+    </div>
+
+    <!-- Myself-->
+    <section class="myself">
+        <h1>About Myself</h1>
```

Comment:
- [ ] Kindly make sure you use on the **About myself** text  a tag different to `h1(e.g.: h2,...)` as your supposed to have only one `h1` tag in your page which is `<h1 class="text"> I’m Muskan</h1>`. Make sure you change everywhere to use this change everywhere you used `h1` except on `<h1 class="text"> I’m Muskan</h1>` for HTML good semantics and accessibility.